src/core/core.rc2

```diff
@@ -91,7 +91,7 @@ BEGIN
   IDSC_IMPORTCONFLICTS2   "Entry on line %d with no group"
   IDSC_IMPORTCONFLICTSX1  "Entry with Group=""%ls"""
   IDSC_IMPORTCONFLICTSX2  "Entry with no group"
-  IDSC_IMPORTABORTED      "Import aborted. Please see report."
+  IDSC_IMPORTABORTED      "Import aborted. See report."
```

Comment:
Maybe keep it here? My thinking is, the word here softens the user frustration from the operation failing.